src/manage.c

```diff
@@ -171,8 +183,16 @@ static void manage_calc_almanac_scores(void)
       /* No almanac or position/time information, give it the benefit of the
        * doubt. */
       acq_prn_param[prn].score[ACQ_HINT_ALMANAC] = 0;
+    } else if (almanac[prn].valid && time_quality > TIME_COARSE &&
+              (abs(almanac[prn].week - t.wn%1024) > MAX_ALM_WEEK_DIFF)) {
```

Comment:
so does this mean that our almanac file and corresponding SBP message should be the full week number?